mrtt-ui/src/components/ProjectDetailsForm.js

```diff
@@ -174,19 +168,20 @@ const ProjectDetailsForm = () => {
         )}
         {/* Countries selector */}
         <FormQuestionDiv>
-          <FormLabel>{questions.countries.question}</FormLabel>
+          <FormLabel htmlFor='countries'>{questions.countries.question}</FormLabel>
           <Controller
             name='countries'
             control={control}
             defaultValue={[]}
             render={({ field }) => (
               <Autocomplete
                 {...field}
+                isOptionEqualToValue={(option, value) => option.name === value.name}
```

Comment:
this was working, but there were warnings when populating from api. Nice convenience feature for mui!

Reply:
are you still seeing those warning @mmnoo , i'm actually not seeing them on this branch, but seeing them on develop 🤔 (the mui errors for countries)

Reply:
looks like this line fixes it